fido-device-onboard.spec

```diff
@@ -0,0 +1,133 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global forgeurl https://github.com/fedora-iot/fido-device-onboard-rs
+
+%forgemeta
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs
+Source:         %{url}/archive/%{commit}/%{name}-rs-%{commit}.tar.gz
```

Comment:
Commit isn't defined anywhere, we should probably just use the version nomenclature everywhere

Reply:
the Makefile defines it - downstream will add it, this is the very same osbuild/osbuild-composer does so you can just git clone this and build a fully functional RPM